Extensions/ArtifactEngine/E2ETests/fileshare.e2e.ts

```diff
@@ -48,5 +49,49 @@ describe('E2E Tests', () => {
                     throw error;
                 });
         });
+
+        var runWindowsBasedTest = process.platform == 'win32' ? it : it.skip;
+        runWindowsBasedTest('should be able to incrementally download the build artifact from fileshare', function (done) {
+            this.timeout(15000);
+            let processor = new engine.ArtifactEngine();
+
+            let processorOptions = new engine.ArtifactEngineOptions();
+            processorOptions.itemPattern = "fileshareWithMultipleFiles\\**";
+            processorOptions.artifactCacheKey = "default_Collection.123.2048.artifactName";
+            processorOptions.artifactCacheDirectory = path.join(nconf.get('CACHE'));
+            processorOptions.parallelProcessingLimit = 8;
+            processorOptions.retryIntervalInSeconds = 2;
+            processorOptions.retryLimit = 2;
+            processorOptions.verbose = true;
+
+            var itemsUrl1 = "C:/vsts-agent/_layout/_work/9/s/fileshareWithMultipleFiles";
```

Comment:
Shouldn't itemsUrl1 and itemsUrl2 be fileshare paths? You can store the test data at the location used in other tests.

Reply:
yes but i didn't had access to that vscsstor so Omesh said to leave it this way

Reply:
we should resolve this before merging the PR by taking gyan's help but first we need to get this PR in a clean state.